sdk/python/feast/infra/offline_stores/redshift.py

```diff
@@ -58,3 +100,61 @@ def get_historical_features(
         project: str,
     ) -> RetrievalJob:
         pass
+
+
+class RedshiftRetrievalJob(RetrievalJob):
+    def __init__(self, query: str, redshift_client, s3_resource, config: RepoConfig):
+        self.query = query
+        self._redshift_client = redshift_client
+        self._s3_resource = s3_resource
+        self._config = config
+        self._s3_path = (
+            self._config.offline_store.s3_path + "/unload/" + str(uuid.uuid4())
+        )
+
+    def to_df(self) -> pd.DataFrame:
```

Comment:
We should definitely add docstrings to all user facing methods.

Reply:
to_df and to_arrow are already documented in the parent class (RetrievalJob). I didn't want to duplicate the docstrings.